internal/entity/activity.go

```diff
@@ -36,32 +34,30 @@ var AllActivityStatusValues = []string{
 }
 
 type Activity struct {
+	Metadata
 	Id        int64               `json:"id"`
 	Status    ActivityStatusValue `json:"status"`
 	Service   *Service            `json:"service,omitempty"`
 	Issues    []Issue             `json:"issues,omitempty"`
 	Evidences []Evidence          `json:"evidences,omitempty"`
-	CreatedAt time.Time           `json:"created_at"`
-	DeletedAt time.Time           `json:"deleted_at,omitempty"`
-	UpdatedAt time.Time           `json:"updated_at"`
 }
 
 func (a *Activity) GetId() int64 {
 	return a.Id
 }
 
 type ActivityHasIssue struct {
-	ActivityId int64     `json:"activity_id"`
-	IssueId    int64     `json:"issue_id"`
-	CreatedAt  time.Time `json:"created_at"`
-	DeletedAt  time.Time `json:"deleted_at,omitempty"`
-	UpdatedAt  time.Time `json:"updated_at"`
+	Metadata
+	ActivityId int64 `json:"activity_id"`
+	IssueId    int64 `json:"issue_id"`
 }
 
 type ActivityAggregations struct {
+	Metadata
 }
 
 type ActivityFilter struct {
+	Metadata
```

Comment:
The filter structs aren't stored in the database, they are used to filter the data in the database. So we don't need the Metadata here